### DISCUSSION TITLE
Please, what's the difference between them

### DISCUSSION THREAD

**Discussion by sangoi-exe** [kohya-ss/sd-scripts#369] [Q&A] [answered]
Between LoRA finetune and LoRA dreambooth?

Also, is there some guide with explanation about the parameters?

**Comment by BarfingLemurs** [comment, marked as answer]
@DevArqSangoi dreambooth training is a variant of finetuning, and finetuning models is a standard machine/deep learning procedure.
Dreambooth training usually intends to focus on adding your training pictures to a word, where you could call a picture of a unique face or some special object maybe not found in the model. Example prompt during inference: `A smiling zwx man`

Finetuning models is usually more involved, you will need to describe each individual training picture with words. This process can be done manually, however. some choose to use machine learning models to tag their images, such as clip, or blip.

The text descriptions for related training images are put in a .json and linked with your training pictures.

The finetuning may allow you to call much more items than dreambooth, which may be limited to 3 or so.
The difficulty with finetuning, may be that clip or blip does a lengthy description of your unique characters, without assigning names.

You would have to call your specific character by using the specific text blip/clip provided, which in my opinion, make it not worth using, its not easy breezy prompting magic.

Finally, training LoRAs are considered training lighter VRAM versions of finetuning. you can do both dreambooth and finetuning. LoRA quality is still great, so you could use it if you don't have much vram.

**Reply by sangoi-exe** [reply]
Thank you very much!!
